### PR TITLE
Add Nethesis to vendors

### PR DESCRIPTION
**Pull request by Stell0**
https://github.com/nethesis/dev/issues/5956

### CODE REVIEW

**Comment by Stell0** [comment]
Update NP5 template and macros with https://github.com/nethesis/tancredi/pull/208 changes when it is merged

**Review by DavidePrincipi — COMMENTED** [review]
Can we use a different name pattern for the template files?

X3 and X5 recall the _model_ name that obey to marketing rules: they are not a solid base for the template name :)

File: data/templates/fallback.tmpl [hunk 1]
```diff
@@ -11,9 +11,9 @@
 {%- elseif provisioning_user_agent starts with 'Maxwell' -%}
 {% include 'gigaset-Maxwell.tmpl' %}
 {%- elseif provisioning_user_agent starts with 'Nethesis NP-X3' -%}
-{% include 'nethesis-NPX3.tmpl' %}
+{% include 'nethesis-V1.tmpl' %}
 {%- elseif provisioning_user_agent starts with 'Nethesis NP-X5' -%}
-{% include 'nethesis-NPX5.tmpl' %}
+{% include 'nethesis-V5.tmpl' %}
```
Comment (DavidePrincipi) [inline data/templates/fallback.tmpl:16]:
```suggestion
{% include 'nethesis-V2.tmpl' %}
```

Comment (Stell0) [reply data/templates/fallback.tmpl]: sniper!

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**